cli/main.c

```diff
@@ -131,14 +131,26 @@ PkStringPtr loadScript(PKVM* vm, const char* path) {
 
 int main(int argc, char** argv) {
 
-  const char* usage = "usage: pocket [-c cmd | file]\n";
-
-  // TODO: implement arg parse, REPL.
-
-  //if (argc < 2) {
-  //  printf("%s\n%s", notice, help);
-  //  return 0;
-  //}
+  const char* usage =
+    "usage: pocket [-c cmd | file]\n\n"
+    "The following options are available:\n\n"
+    "  -c cmd    :    program passed in as string (terminates option list)\n"
+    "  file      :    program read from script file\n"
+    "  --version :    prints the version of pocketlang\n"
+    "  --help    :    prints this help message\n";
+
+  if (argc > 1) {
+    if (strcmp(argv[1], "--help") == 0) {
+      printf("%s", usage);
+      return 0;
+    } else if (strcmp(argv[1], "--version") == 0) {
+      printf("Version: %s\n", PK_VERSION_STRING);
+      return 0;
+    } else {
+      printf("Invalid option flag. Use --help for more.\n");
```

Comment:
We can't run `pocket file.pk` anymore since `file.pk` here isn't a valid option.

Reply:
Ooh, and the error messages should be printed to `stderr` in this case.

Reply:
Ooh, and also error messages should be written to `stderr` in this case.